google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb

```diff
@@ -750,6 +750,10 @@ def tables token: nil, max: nil
         #   * `append` - BigQuery appends the data to the table.
         #   * `empty` - A 'duplicate' error is returned in the job result if the
         #     table exists and contains data.
+        # @param [Boolean] dry_run If set to true, BigQuery doesn't run the job.
```

Comment:
I expected to see this optional argument listed at the end of the arguments, not near the top. Is this argument more important than those below it?

Reply:
Yes, I think so.